### PR TITLE
ZENKO-3386: enhance UX on "Add New Storage Location" form

### PR DESCRIPTION
**Pull request by theocerutti**

### CODE REVIEW

**Review by Cuervino — APPROVED** [review]
top!

**Review by terrydervaux — APPROVED** [review]
LGTM

**Review by nicolas2bert — APPROVED** [review]

**Comment by theocerutti** [comment]
@bert-e approve

**Comment by terrydervaux** [comment]
@bert-e ping

File: src/react/account/details/__tests__/Locations.test.jsx [hunk 1]
```diff
@@ -150,7 +150,7 @@ describe('Locations', () => {
         const thirdRowColumns = thirdRow.find(T.Cell).map(column => column.text());
         expect(thirdRowColumns.length).toEqual(nbrOfColumnsExpected);
         expect(thirdRowColumns[0]).toEqual('location-file');
-        expect(thirdRowColumns[1]).toEqual('Zenko Local Filesystem');
```
Comment (nicolas2bert) [inline src/react/account/details/__tests__/Locations.test.jsx:153]: Sorry for being the "bad cop" here but it seems to be some other places (especially in location creation) where "zenko" is mentioned. Can you please look at it @theocerutti ? Also, we do not need to change it if Zenko appears in a doc file or comments, only for the rendered code. Thank you.

**Review by nicolas2bert — CHANGES_REQUESTED** [review]

File: src/react/account/details/__tests__/Locations.test.jsx (hunk 1 above)
Comment (theocerutti) [reply src/react/account/details/__tests__/Locations.test.jsx:153]: done

**Review by nicolas2bert — APPROVED** [review]

**Comment by theocerutti** [comment]
@bert-e approve

**Comment by theocerutti** [comment]
@bert-e approve

**Comment by nicolas2bert** [comment]
@bert-e reset

**Pull request closed without merging**